cardano-cli/src/Cardano/CLI/Shelley/Orphans.hs

```diff
@@ -104,82 +106,64 @@ deriving newtype instance ToJSON BlockNo
-deriving newtype instance ToJSON (Ledger.Stake StandardAllegra)
-deriving newtype instance ToJSON (Ledger.Stake StandardMary)
+deriving newtype instance ToJSON (Ledger.Stake StandardCrypto)
 
 deriving anyclass instance ToJSON (Ledger.GenDelegs StandardCrypto)
 deriving anyclass instance ToJSON (Ledger.IndividualPoolStake StandardCrypto)
+deriving anyclass instance ToJSON (Ledger.BlocksMade StandardCrypto)
 
 deriving anyclass instance ToJSON (Ledger.ProposedPPUpdates StandardShelley)
 deriving anyclass instance ToJSON (Ledger.PPUPState StandardShelley)
-deriving anyclass instance ToJSON (Ledger.BlocksMade StandardShelley)
 
 deriving anyclass instance ToJSON (Ledger.ProposedPPUpdates StandardAllegra)
 deriving anyclass instance ToJSON (Ledger.PPUPState StandardAllegra)
-deriving anyclass instance ToJSON (Ledger.BlocksMade StandardAllegra)
 
 deriving anyclass instance ToJSON (Ledger.ProposedPPUpdates StandardMary)
 deriving anyclass instance ToJSON (Ledger.PPUPState StandardMary)
-deriving anyclass instance ToJSON (Ledger.BlocksMade StandardMary)
 
 deriving instance ToJSON Ledger.Ptr
 deriving instance ToJSON Ledger.AccountState
 
-deriving instance ToJSON (Ledger.DPState StandardShelley)
-deriving instance ToJSON (Ledger.DState StandardShelley)
-deriving instance ToJSON (Ledger.InstantaneousRewards StandardShelley)
-deriving instance ToJSON (Ledger.SnapShot StandardShelley)
-deriving instance ToJSON (Ledger.SnapShots StandardShelley)
-deriving instance ToJSON (Ledger.NonMyopic StandardShelley)
+deriving instance ToJSON (Ledger.DPState StandardCrypto)
+deriving instance ToJSON (Ledger.DState StandardCrypto)
+deriving instance ToJSON (Ledger.InstantaneousRewards StandardCrypto)
+deriving instance ToJSON (Ledger.NonMyopic StandardCrypto)
+deriving instance ToJSON (Ledger.PState StandardCrypto)
+deriving instance ToJSON (Ledger.RewardUpdate StandardCrypto)
+deriving instance ToJSON (Ledger.SnapShot StandardCrypto)
+deriving instance ToJSON (Ledger.SnapShots StandardCrypto)
+deriving instance ToJSON (Ledger.StakeReference StandardCrypto)
+
 deriving instance ToJSON (Ledger.LedgerState StandardShelley)
 deriving instance ToJSON (Ledger.EpochState StandardShelley)
-deriving instance ToJSON (Ledger.RewardUpdate StandardShelley)
 deriving instance ToJSON (Ledger.NewEpochState StandardShelley)
 deriving instance ToJSON (Ledger.PParams' StrictMaybe StandardShelley)
-deriving instance ToJSON (Ledger.PState StandardShelley)
-deriving instance ToJSON (Ledger.StakeReference StandardShelley)
 deriving instance ToJSON (Ledger.UTxOState StandardShelley)
 
-deriving instance ToJSON (Ledger.DPState StandardAllegra)
-deriving instance ToJSON (Ledger.DState StandardAllegra)
-deriving instance ToJSON (Ledger.InstantaneousRewards StandardAllegra)
-deriving instance ToJSON (Ledger.SnapShot StandardAllegra)
-deriving instance ToJSON (Ledger.SnapShots StandardAllegra)
-deriving instance ToJSON (Ledger.NonMyopic StandardAllegra)
```

Comment:
A bunch of these are now parameterised by the crypto, so only a single instance is needed instead of one for each era.